test/model.indexes.test.js

```diff
@@ -83,33 +83,28 @@ describe('model', function() {
       const UserModel = db.model('Test', User);
       let assertions = 0;
 
-      UserModel.on('index', function() {
-        UserModel.collection.getIndexes(function(err, indexes) {
-          assert.ifError(err);
+      await UserModel.init();
 
-          function iter(index) {
-            if (index[0] === 'name') {
-              assertions++;
-            }
-            if (index[0] === 'blogposts._id') {
-              assertions++;
-            }
-            if (index[0] === 'blogposts.title') {
-              assertions++;
-            }
-          }
+      const mongoIndexes = await UserModel.collection.getIndexes();
```

Comment:
Looks like `getIndexes()` resolves to an object now, `Object.values(await UserModel.collection.getIndexes())` should be what you're looking for.

Reply:
Thanks, I'll take a look at those failing tests sometime during the weekend, then we can merge and I can work on another batch on another PR.